healpy/sphtfunc.py

```diff
@@ -972,9 +979,9 @@ def pixwin(nside, pol=False):
     pw = pf.getdata(fname)
     pw_temp, pw_pol = pw.field(0), pw.field(1)
     if pol:
-        return pw_temp, pw_pol
+        return pw_temp[: lmax + 1], pw_pol[: lmax + 1]
```

Comment:
the spacing here looks strange, is `black` ok with this?

Reply:
Yes, this is black's default behavior. This is cleary different to what PEP8 says (https://www.python.org/dev/peps/pep-0008/#whitespace-in-expressions-and-statements).